pkg/gatherers/conditional/gathering_rule.schema.json

```diff
@@ -74,6 +74,40 @@
                                 }
                             }
                         }
+                    },
+                    {
+                        "type": "object",
+                        "title": "ConditionWithParams",
+                        "description": "cluster_version_matches condition",
+                        "required": [
+                            "type",
+                            "params"
```

Comment:
shouldn't this be `cluster_version_matches` ? And same below

Reply:
@tremes Could you please clarify what you mean? Because as far as I can see, the description is consistent with the `alert_is_firing` condition description field and the `required` fields match the fields defined below. Am I missing something?

Reply:
AFAIK the validation was failing for me. Maybe I was doing something wrong or just missed something, but I think the naming here has to match the naming in https://github.com/openshift/insights-operator/pull/524/files#diff-4e4038b9223da9895b1e6eda486f7d60a3453f9e39e36aefc3d35a3c80d14c78R7 otherwise the validation fails.